app/controllers/api/users.php

```diff
@@ -1784,7 +1784,7 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             throw new Exception(Exception::USER_NOT_FOUND);
         }
 
-        $secret = Auth::codeGenerator();
+        $secret = Auth::tokenGenerator(Auth::TOKEN_LENGTH_SESSION);
```

Comment:
@loks0n, do you remember why the secret was created using `Auth::codeGenerator();` which creates a 6 character code?